ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -592,20 +594,21 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
-				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				CURRENT_PROJECT_VERSION = 45;
```

Comment:
no env var?

Reply:
Good question, since this is in the IOS section of the application I'll ask @coeneivan about it.

Reply:
Reverting this so we can use env vars again